config/enrichments/07_lowercase.conf

```diff
@@ -47,13 +46,16 @@ filter {
         "client.bytes",
         "client.domain",
         "client.geo.city_name",
+        "client.geo.continent_code",
```

Comment:
These are standard uppercase. I'm not sure if we should lowercase these.
@brian-grabau  should we lowercase such fields? e.g.
geo.continent_code
geo.postal_code
geo.timezone

Reply:
If it is safe to assume ECS is America-centric, postal codes are always numbers. In other countries that have postal codes, are would the postal code "203foo" point to a different location than "203FOO?"

The examples in ECS for geo.continent_code is "NA" and the timezone is "America/Argentina/Buenos_Aires." Since there isn't a substantive difference in "na" vs "NA" (there isn't another continent called "na"), I can't think of a harmful reason to lowercase them. I think one negative consequence of leaving them uppercase is that other instances of regional data - for example, in host domains - are already lowercased.